examples/graphql-subscriptions/graphql_subscriptions.md

```diff
@@ -0,0 +1,21 @@
+# Subscriptions
+
+A resource function with the `subscribe` accessor inside a GraphQL service
+represents a field in the root `Subscription` object type. Therefore, if a
+resource function with the `subscribe` accessor is present inside the
+Ballerina GraphQL service, the auto-generated schema will have a `Subscription`
+type.
+
+Each resource function with the `subscribe` accessor in the service will
+be added as a field of the `Subscription` type. The field name will be the
+resource function name, and the field type will be the constraint type of
+the stream returned from the resource function.
+
+For more information on the underlying package, see the
+[GraphQL package](https://docs.central.ballerina.io/ballerina/graphql/latest/).
+
+::: code graphql_subscriptions.bal :::
```

Comment:
```suggestion
::: code graphql_subscriptions.bal :::
```
You need to add an instruction line before the `.out` files. Check the samples given as the guideline and fix all the occurrences accordingly.